stable/influxdb/templates/post-install-create-db.yaml

```diff
@@ -0,0 +1,43 @@
+{{- if .Values.createDatabase.enabled -}}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+  name: {{ template "fullname" . }}-create-db
+  annotations:
+    "helm.sh/hook": post-install
+spec:
+  activeDeadlineSeconds: {{ default 300 .Values.createDatabase.activeDeadlineSeconds }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "fullname" . }}
+        release: "{{ .Release.Name }}"
+    spec:
+      containers:
+      - name: {{ template "fullname" . }}-create-db
+        image: "{{ .Values.createDatabase.image }}"
```

Comment:
Why not just use the `influxdb:{{ version }}` image that gets pulled to create the database? Its a really small `alpine` container that has the `influx` cli already installed. It would make this call really easy and remove the need for another container image and the associated configuration.